src/models/RecommenderVersion.py

```diff
@@ -1,6 +1,12 @@
 import enum
 
 
+# Version | Question  | Answers
+# 1       | random    | random
+# 2       | nerandom  | random
+# 3       | random    | nerandom
+# 4       | nerandom  | nerandom
```

Comment:
Nerandom pojdem nahrad za "trained", ako je to uz spravene vo zvysku appky. Pojdem trained je vhodnejsi, pretoze opisuje, ze tam nastal nejaky proces zahrnujuci nejake data a je zauzivany v terminologii. Samozrejme, ak prides na nejaky iny lepsie znejuci pojem, go for it. Ale prezentovat nieco ako "nerandom" vyzera dobre na tabuli, ked ludia ledva chapu, co sa ide diat, no zaroven to nie je uplne terminus technicus.